src/traits/snark.rs

```diff
@@ -55,6 +55,43 @@ pub trait RelaxedR1CSSNARKTrait<E: Engine>:
   fn verify(&self, vk: &Self::VerifierKey, U: &RelaxedR1CSInstance<E>) -> Result<(), NovaError>;
 }
 
+/// A trait that defines the behavior of a `zkSNARK` to prove knowledge of satisfying witness to batches of relaxed R1CS instances.
```

Comment:
Nit: It likely makes sense to move this inside supernova module.

Reply:
The two instances are not in the Supernova module, so I would need to move at least their implementation of the trait to the Supernova module. 

Specifically, considering that snark and ppsnark are, respectively, instances of batched_snark and batched_ppsnark at batch_length = 1, the later could subsume the former.